pdr_backend/pdr_dashboard/test/test_callbacks_feeds.py

```diff
@@ -0,0 +1,231 @@
+import json
+import time
+
+from selenium.webdriver.common.by import By
+from pdr_backend.pdr_dashboard.test.resources import (
+    _navigate_to_feeds_page,
+    _assert_table_row_count,
+    start_server_and_wait,
+    _clear_filters,
+    _set_dropdown_and_verify_row_count,
+    _set_input_value_and_submit,
+    _remove_tags,
+    _set_searchbar_value,
+)
+
+
+def _get_table_data_as_json(table):
+    rows = table.find_elements(By.XPATH, ".//tr")
+    header = rows[0].find_elements(By.XPATH, ".//th")
+
+    headers = [_remove_tags(th.text) for th in header]
+
+    table_data = []
+
+    for row in rows[1:]:  # Skip header row
+        cells = row.find_elements(By.XPATH, ".//td")
+        row_data = {headers[i]: _remove_tags(cells[i].text) for i in range(len(cells))}
+        table_data.append(row_data)
+
+    return table_data
+
+
+def _save_table_data_as_json(table, filename):
+    table_data = _get_table_data_as_json(table)
+    with open("pdr_backend/pdr_dashboard/test/" + filename, "w") as f:
```

Comment:
I initially thought the new test resources were fixed. However, I know see what you are doing here. I suggest moving these "volatile" test resources in a specific subfolder and adding them to gitignore.

Keep a clear distinction between the reference and in-process resources (json files).

Reply:
Looks like this function is not used, so we can delete it